README.rst

```diff
@@ -384,6 +384,9 @@ TMT_REBOOT_COUNT
     for requesting the reboot, variables ``REBOOTCOUNT`` and
     ``RSTRNT_REBOOTCOUNT`` contain number of reboots as well.
 
+TMT_RUN_DIR
+    Directory for storing runtime data. By default ``/var/tmp/tmt``.
+
```

Comment:
This section describes variables which are provided to tests during execution. Let's move it to other variables affecting `tmt` behaviour.

Regarding the name: When I saw the pull request summary, my first idea was that this is adding a new variable with the actual `tmt run` workdir under execution. Perhaps we could keep it consistent with the `tmt status` option instead?

    --workdir-root PATH  Path to root directory containing run workdirs.
                         Defaults to '/var/tmp/tmt'.

So it would be `TMT_WORKDIR_ROOT`. What do you think?

Reply:
sure, I am fine with renaming

Reply:
will be renamed

Reply:
The new option is still included in the wrong section. Should go up to variables which affect `tmt` behaviour, not test-related ones.

Reply:
Fixed in decdc9f4.